extensions/3DTILES_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1623 @@
+#### schema.classes
+
+A dictionary, where each key is a class ID and each value is an object defining the class.
+
+* **Type**: `object`
+* **Required**: No
+* **Type of each property**: `class`
+
+<!-- omit in toc -->
+#### schema.enums
+
+A dictionary, where each key is an enum ID and each value is an object defining the values for the enum.
+
+* **Type**: `object`
+* **Required**: No
+* **Type of each property**: `enum`
+
+<!-- omit in toc -->
+#### schema.extensions
+
+* **Type**: `any`
+* **Required**: No
+
+<!-- omit in toc -->
+#### schema.extras
+
+* **Type**: `any`
+* **Required**: No
+
+---------------------------------------
+<a name="reference-class"></a>
+<!-- omit in toc -->
+### Class
+
+A class containing a set of properties.
+
+**`Class` Properties**
+
+|   |Type|Description|Required|
+|---|---|---|---|
+|**name**|`string`|The name of the class, e.g. for display purposes.|No|
+|**description**|`string`|The description of the class.|No|
+|**properties**|`object`|A dictionary, where each key is a property ID and each value is an object defining the property.|No|
+|**extensions**|`any`||No|
+|**extras**|`any`||No|
+
+Additional properties are allowed.
+
+<!-- omit in toc -->
+#### class.name
+
+The name of the class, e.g. for display purposes.
+
+* **Type**: `string`
+* **Required**: No
+* **Minimum Length**`: >= 1`
+
+<!-- omit in toc -->
+#### class.description
+
+The description of the class.
+
+* **Type**: `string`
+* **Required**: No
+* **Minimum Length**`: >= 1`
+
+<!-- omit in toc -->
+#### class.properties
+
+A dictionary, where each key is a property ID and each value is an object defining the property.
+
+* **Type**: `object`
+* **Required**: No
+* **Type of each property**: `class.property`
+
+<!-- omit in toc -->
+#### class.extensions
+
+* **Type**: `any`
+* **Required**: No
+
+<!-- omit in toc -->
+#### class.extras
+
+* **Type**: `any`
+* **Required**: No
+
+---------------------------------------
+<a name="reference-class-property"></a>
+<!-- omit in toc -->
+### Class property
+
+A class property.
+
+**`Class property` Properties**
+
+|   |Type|Description|Required|
+|---|---|---|---|
+|**name**|`string`|The name of the property, e.g. for display purposes.|No|
+|**description**|`string`|The description of the property.|No|
+|**type**|`string`|The property type. If `ENUM` is used, then `enumType` must also be specified. If `ARRAY` is used, then `componentType` must also be specified. `ARRAY` is a fixed-length array when `componentCount` is defined, and variable-length otherwise.| &#10003; Yes|
+|**enumType**|`string`|An enum ID as declared in the `enums` dictionary. This value must be specified when `type` or `componentType` is `ENUM`.|No|
+|**componentType**|`any`|When `type` is `ARRAY` this indicates the type of each component of the array. If `ENUM` is used, then `enumType` must also be specified.|No|
+|**componentCount**|`integer`|The number of components per element for `ARRAY` elements.|No|
```

Comment:
I'm still confused on component vs. element vocab (CC #455 or #456), and now I am more confused. 😄 

An array element can have more than one component just like a vertex attribute can?